packages/marathon/build

```diff
@@ -19,6 +19,8 @@ StartLimitInterval=0
 RestartSec=15
 PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/marathon
```

Comment:
Right. I was just saying we put everything in command-line now, but we would need `EnvironmentFile=-/opt/mesosphere/etc/marathon` if we wanted to specify those in environment variables instead. Migrating from cli flags to env vars can be done as a separate PR.